server/conf/i18n/messages

```diff
@@ -209,6 +229,9 @@ title.displayingOneClient=Displaying 1 client
 # This is the heading above a table that contains the members of the current community-based organization.
 title.orgMembers=Organization members
 
+# Toast message top of the page when viewing an application
```

Comment:
suggestion (blocking):  Same as above, this isn't a toast message, so maybe "Banner at the top of the page when...".  The key would also change from "toast.viewApplication" to "banner.viewApplication".

Reply:
change applied.